NashvilleHistory/www/js/factories/AllPlacesFact.js

```diff
@@ -14,62 +14,65 @@ app.factory("AllPlacesFact", ($q, $http, KeyGetter)=>{
     return $q((resolve, reject)=>{
       $http.get(`https://data.nashville.gov/resource/m4hn-ihe4.json?$$app_token=${NashvilleGovAppToken}`)
       .success((data)=>{
-        resolve(data);
+        resolve(data.map(marker => {
+          marker.title = parseSpecialCharacters(marker.title)
+          return marker;
+        }).filter((marker) => {
+          if (marker.longitude) {
```

Comment:
toss out any markers that don't come with a longitude and latitude, they're no good for us